CRM/Upgrade/Incremental/sql/4.6.17.mysql.tpl

```diff
@@ -1 +1,4 @@
 {* file to handle db changes in 4.6.17 during upgrade *}
+-- CRM-18516 Convert the date fields relating to group caching and acl caching timestamp
```

Comment:
Given that 4.6.17 is now out then this should probably be putting this into 4.6.18.mysql.tpl